src/ui/components/results/universalresultscomponent.js

```diff
@@ -51,11 +51,24 @@ export default class UniversalResultsComponent extends Component {
     return true;
   }
 
+  mount (container) {
+    const searchState = this.getState('searchState');
+    const camelCase = (str) => {
+      return str
+        .split('-')
+        .reduce((a, b) => a + b.charAt(0).toUpperCase() + b.slice(1));
+    };
+    Object.keys(SearchStates).forEach(state => super.removeContainerClass(`yxt-Results--${camelCase(state)}`));
+    super.addContainerClass(`yxt-Results--${camelCase(searchState)}`);
```

Comment:
is there a reason we're using `super` specifically here (and above) instead of `this`?

Reply:
Oops, let me change that